ts/features/pn/components/MessageDetails.tsx

```diff
@@ -38,36 +47,50 @@ export const MessageDetails = ({ message, serviceId }: MessageDetailsProps) => {
   const attachmentList = SEP.left(partitionedAttachments);
 
   return (
-    <SafeAreaView edges={["bottom"]} style={IOStyles.flex}>
-      <ScrollView>
-        <ContentWrapper>
-          <MessageDetailsHeader
-            serviceId={serviceId}
-            subject={message.subject}
-            createdAt={message.created_at}
-          >
+    <ScrollView
+      contentContainerStyle={{
+        paddingBottom: IOStyles.footer.paddingBottom + safeAreaInsets.bottom
+      }}
+    >
+      <ContentWrapper>
+        <MessageDetailsHeader
+          serviceId={serviceId}
+          subject={message.subject}
+          createdAt={message.created_at}
+        >
+          <MessageDetailsTagBox>
+            <Tag
+              text={I18n.t("features.pn.details.badge.legalValue")}
+              variant="legalMessage"
+            />
+          </MessageDetailsTagBox>
+          {attachmentList.length > 0 && (
             <MessageDetailsTagBox>
               <Tag
-                text={I18n.t("features.pn.details.badge.legalValue")}
-                variant="legalMessage"
+                variant="attachment"
+                testID="attachment-tag"
+                iconAccessibilityLabel={I18n.t(
+                  "messageDetails.accessibilityAttachmentIcon"
+                )}
               />
             </MessageDetailsTagBox>
-            {attachmentList.length > 0 && (
-              <MessageDetailsTagBox>
-                <Tag
-                  variant="attachment"
-                  testID="attachment-tag"
-                  iconAccessibilityLabel={I18n.t(
-                    "messageDetails.accessibilityAttachmentIcon"
-                  )}
-                />
-              </MessageDetailsTagBox>
-            )}
-            <VSpacer size={8} />
-          </MessageDetailsHeader>
-          <MessageDetailsContent abstract={message.abstract} />
-        </ContentWrapper>
-      </ScrollView>
-    </SafeAreaView>
+          )}
+          <VSpacer size={8} />
+        </MessageDetailsHeader>
+        <MessageDetailsContent abstract={message.abstract} />
+        <VSpacer />
+        <MessageDetailsAttachments
+          disabled={message.isCancelled}
+          messageId={messageId}
+          isPN
+        />
+        <VSpacer />
```

Comment:
```suggestion
        <VSpacer size={16} />
        <MessageDetailsAttachments
          disabled={message.isCancelled}
          messageId={messageId}
          isPN
        />
        <VSpacer size={16} />
```
Does not change anything as of right now but I think being more explicit would make it a tad bit cleaner; what do you think?

Reply:
Not sure about it, @dmnplb @CrisTofani @shadowsheep1 @mastro993 @adelloste what is your opinion on this matter? We either:
- know that, by default, a <VSpacer /> is 16 points tall and if the default changes in the future, it will beapplied automatically. As a direct consequence of this, we do not explicitly set its size when it is equal to 16 points;
- favour readability/explicitability over hiding (the default) and always resort to setting a size value (in this case, it is also worth removing the default setting from the DS package, for consistency). The downside is that if the default changes in the future, we will have to manually apply it to all code

Reply:
Right at this moment if we consider to remove/change the default we should already properly check all the code.

BTW we're not considering to change the default or remove it as a current activity, and as @Vangaorth we can use it's implicit default as code readability

Reply:
I agree with @mastro993.

Reply:
Ok thanks everybody, I applied what @forrest57 and @mastro993 suggested